### DISCUSSION TITLE
建议把社区翻译好的中文挂在官网方便国人阅读

### DISCUSSION THREAD

**Discussion by orjuly** [apache/skywalking#7490] [General]
个人开源走到apache开源不易，期望大家一起维护中文文档！

**Comment by wu-sheng** [comment]
If you want to maintain Chinese document, it is there. But reality, no one wants to do that.

**Reply by wu-sheng** [reply]
https://github.com/SkyAPM/document-cn-translation-of-skywalking
It is there for years. No one continues on that.
